util/chunk/chunk.go

```diff
@@ -706,3 +706,26 @@ func (c *Chunk) ToString(ft []*types.FieldType) string {
 	}
 	return string(buf)
 }
+
+// AppendRows appends multiple rows  to the chunk.
+func (c *Chunk) AppendRows(rows []Row) {
+	c.AppendPartialRows(rows)
+	c.numVirtualRows += len(rows)
+}
+
+// AppendPartialRows appends multiple row to the chunk.
+func (c *Chunk) AppendPartialRows(rows []Row) {
```

Comment:
1. Should also deliver the `colOff` to the function?
2. Should also consider `c.sel` ?

Reply:
the first it is considered , colOff the sepcial cond ，I test not be used, so i delete it 

so  add it as AppendPartialRow